lib/Core/Executor.cpp

```diff
@@ -520,6 +518,7 @@ Executor::~Executor() {
 }
 
 /***/
+bool enteredDevMain = false;
```

Comment:
you can set this to true to test. It set it to false before I understood why I was getting undefined mem error before reaching the main function... then I realized I needed this extra `klee_ignore_undefined` function + uclibc patches.